### PR TITLE
adding travel info to the ticket page

### PR DESCRIPTION
**Pull request by leftieFriele**
added the travel info, with a link from the tickets page.

### CODE REVIEW

File: views/tickets.hbs [hunk 1]
```diff
@@ -35,9 +35,10 @@
       <li>full <strong>dinner</strong> at the party Thursday</li>
       <li>a decent amount of <strong>free beverages</strong> during the party</li>
     </ul>
+    <p class="mbxl">Not sure how to best travel to Oslo? Checkout our <a class="link" href="/travel">travel guide</a></p>
```
Comment (Andersos) [inline views/tickets.hbs]: Add a periode

File: views/tickets.hbs [hunk 2]
```diff
@@ -35,9 +35,10 @@
...
     <p class="mbs">You can read more on our blog about the ticket sales process:</p>
-    <p class="mbxl"><a class="link"href="https://webrebels.tumblr.com/post/139069607543/ticket-sale">About the 2016 ticket sales</a>
-        <a class="link" href="https://webrebels.tumblr.com/post/111637996063/gone-in-less-than-60-seconds">“Gone in less than 60 seconds”</a>
+    <p class="mbxl"><a href="https://webrebels.tumblr.com/post/139069607543/ticket-sale">About the 2016 ticket sales</a> &amp;
```
Comment (Andersos) [inline views/tickets.hbs]: The two links look like button. I don't feel the amp is needed.

File: views/tickets.hbs (hunk 1 above)
Comment (leftieFriele) [inline views/tickets.hbs]: :+1: done

**Comment by Andersos** [comment]
LGTM.

**Pull request merged**